optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/ProblemBenchmarkResult.java

```diff
@@ -453,9 +453,9 @@ public int hashCode() {
                             it.remove();
                         }
                     }
-                    newResult.entityCount = ConfigUtils.mergeProperty(oldResult.entityCount, newResult.entityCount);
-                    newResult.variableCount = ConfigUtils.mergeProperty(oldResult.variableCount, newResult.variableCount);
-                    newResult.problemScale = ConfigUtils.mergeProperty(oldResult.problemScale, newResult.problemScale);
+                    newResult.entityCount = ConfigUtils.meldProperty(oldResult.entityCount, newResult.entityCount);
```

Comment:
It was a bug when merging a problem benchmark result with only failed singles. In SingleBenchmarkRunner, `registerScale()` never gets called (for example, the others too). It's version agnostic. So you fix the error, rerun benchmarker, and want to merge those reports. That's when it happens.

Yes, we can leave it to become null, but we know the problem scale from one of them, why not use it? It hasn't changed, the dataset is the same (see `fileToNewResultMap`.. of course, assuming the user didn't change the dataset between runs).